docs/upgrade.html

```diff
@@ -40,13 +40,18 @@ <h5><a id="upgrade_390_notable" href="#upgrade_390_notable">Notable changes in 3
             For example, this could be done via the <code>KAFKA_OPTS</code> environment variable like follows: <code>export KAFKA_OPTS="-DZstdTempFolder=/opt/kafka/tmp -Dorg.xerial.snappy.tempdir=/opt/kafka/tmp"</code>.
 	    This is a known issue for version 3.8.0 as well.
         </li>
-        <li>In KRaft mode, the tiered storage feature can be dynamically disabled and then re-enabled on topic level.
-            See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-950%3A++Tiered+Storage+Disablement">KIP-950</a> for more details.</li>
-        <li>Tiered storage quota is implemented. Users can set an "upper bound" on the rate at which logs are copied/read to/from the remote storage.
-            See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-956+Tiered+Storage+Quotas">KIP-956</a> for more details.</li>
         <li><code>unclean.leader.election.enable</code> config is supported in KRaft. Compared with ZK mode, there is one behavior change in KRaft mode
             when dynamically enabling <code>unclean.leader.election.enable</code> config. Please check
             <a href="https://kafka.apache.org/documentation/#brokerconfigs_unclean.leader.election.enable">here</a> for more details.</li>
+        <li>Tiered storage is now a production ready feature
+            <ul>
```

Comment:
Good to explicitly callout that the below features are added in this release like below. 

```
            The below enhancements are added in this release.
```